src/main/scala/org/novetta/zoo/services/office_meta/service.py

```diff
@@ -0,0 +1,137 @@
+import tornado
+from tornado import web, httpserver, ioloop
+import tornado.options
+from tornado.options import define, options
+
+import traceback
+import os
+from os import path
+
+from office_meta      import OfficeParser, OfficeMetaError
+from library.services import ServiceResultSet, ServiceConfig
+
+import binascii
+
+
+# get config and init Tornado
+Config = ServiceConfig("./service.conf")
+define("port", default=int(Config.settings.port,10), help="port to run", type=int)
```

Comment:
please remove default port 10. Would rather have the setting match the docker file or ideally just fail.